packages/components/src/components/form/Textarea/index.tsx

```diff
@@ -13,134 +15,83 @@ const Wrapper = styled.div`
-        `}
+    :hover {
+        border-color: ${({ state, theme }) =>
+            state
+                ? darken(theme.HOVER_DARKEN_FILTER, getStateColor(state, theme))
+                : darken(theme.HOVER_DARKEN_FILTER, theme.STROKE_GREY)};
+    }
 
-    ${props =>
-        props.monospace &&
+    :focus {
+        border-color: ${({ state, theme }) =>
+            state
+                ? darken(theme.HOVER_DARKEN_FILTER, getStateColor(state, theme))
+                : theme.TYPE_LIGHT_GREY};
+    }
+
+    ${({ disabled }) =>
+        disabled &&
         css`
-            font-variant-numeric: slashed-zero tabular-nums;
+            background: ${({ theme }) => theme.BG_GREY};
+            box-shadow: none;
+            color: ${({ theme }) => theme.TYPE_DARK_GREY};
+            pointer-events: none;
+            cursor: default;
         `}
 `;
 
-const BottomText = styled.span<StyledTextareaProps>`
+const BottomText = styled.span<Pick<TextareaProps, 'state'>>`
     padding: 10px 10px 0 10px;
     min-height: 27px;
-    font-size: ${variables.FONT_SIZE.TINY};
-    color: ${props => getStateColor(props.state, props.theme)};
-`;
-
-const TooltipAction = styled.div<{ action: React.ReactNode }>`
-    display: ${props => (props.action ? 'flex' : 'none')};
-    align-items: center;
-    margin: 0px 10px;
-    padding: 0 14px;
-    position: absolute;
-    background: black;
-    bottom: -25px;
-    color: ${props => props.theme.BG_WHITE};
-    border-radius: 5px;
-    z-index: 10002;
-    transform: translate(-1px, -1px);
-`;
-
-const ArrowUp = styled.div`
-    position: absolute;
-    top: -9px;
-    left: 12px;
-    width: 0;
-    height: 0;
-    border-left: 9px solid transparent;
-    border-right: 9px solid transparent;
-    border-bottom: 9px solid black;
-    z-index: 10001;
-`;
-
-const Label = styled.div`
-    display: flex;
-    min-height: 32px;
-    justify-content: space-between;
-`;
-
-const Left = styled.label`
-    display: block;
-    font-size: ${variables.FONT_SIZE.NORMAL};
-    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
-    padding: 0 0 12px 0;
-    color: ${props => props.theme.TYPE_DARK_GREY};
-`;
-
-const Right = styled.div`
-    display: flex;
-`;
-
-const VisibleRightLabel = styled.div`
-    padding-left: 5px;
+    font-size: ${FONT_SIZE.TINY};
+    color: ${({ state, theme }) => getStateColor(state, theme)};
 `;
 
 const LabelAddon = styled.div``;
 
-type BaseTextareaProps = React.TextareaHTMLAttributes<HTMLTextAreaElement>;
-
-// TODO: proper types for wrapperProps (should be same as React.HTMLAttributes<HTMLDivElement>)
-export interface TextareaProps extends StyledTextareaProps {
+export interface TextareaProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
     isDisabled?: boolean;
     label?: React.ReactNode;
     labelAddon?: React.ReactNode;
     labelRight?: React.ReactNode;
     innerRef?: React.Ref<HTMLTextAreaElement>;
     bottomText?: React.ReactNode;
+    width?: any;
```

Comment:
why any?

Reply:
Should be `number` ofc, missed that when moved the props :)